bundle/manifests/cert-manager-operator.clusterserviceversion.yaml

```diff
@@ -673,6 +673,7 @@ spec:
                 - --v=$(OPERATOR_LOG_LEVEL)
                 - --trusted-ca-configmap=$(TRUSTED_CA_CONFIGMAP_NAME)
                 - --cloud-credentials-secret=$(CLOUD_CREDENTIALS_SECRET_NAME)
+                - --addon-features=$(ADDON_FEATURES)
```

Comment:
How does this distinguish between unsupported vs supported add-ons?

Reply:
As discussed, updated arg name to `--unsupported-addon-features`,
and additionally in operator docs the tech preview related warning will be presented to users who wish to enable IstioCSR addon feature.